packages/asset-swapper/src/utils/market_operation_utils/index.ts

```diff
@@ -176,9 +176,17 @@ export class MarketOperationUtils {
             ? this._sampler.getBalancerSellQuotesOffChainAsync(makerToken, takerToken, sampleAmounts)
             : Promise.resolve([]);
 
-        const offChainBancorPromise = _opts.excludedSources.includes(ERC20BridgeSource.Bancor)
-            ? Promise.resolve([])
-            : this._sampler.getBancorSellQuotesOffChainAsync(makerToken, takerToken, [takerAmount]);
+        const bntToken =
+            this._sampler.bancorService &&
+            (await this._sampler.bancorService.getBancorTokenAddressAsync()).toLowerCase();
```

Comment:
Is this forever cached? I imagine we might be able to hard code it anyway, would it ever change?

Reply:
ya, this is cached. I made some changes to make it non-async.